src/modules/components/odsModulesProvider/odsModulesProvider.tsx

```diff
@@ -31,18 +32,25 @@ export interface IOdsModulesProviderProps {
      * @default defaultQueryClient
      */
     queryClient?: QueryClient;
+    /**
+     * Values for the OdsCoreProvider context.
+     * @see IOdsCoreContext
+     */
+    coreProviderValues?: IOdsCoreProviderProps['values'];
```

Comment:
Yes, as you mentioned, the idea is that the `OdsModulesProvider` handle everything included on the `OdsCoreProvider` so that applications don't need to use both providers but only one. Even though the core-provider only handles the Link & Image components, I would still keep a generic name as core-values so that, if we add more properties to the core context, we don't need to update the OdsModulesProvider as well and change the property name